### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by tryggvigy**
## Description
 Add a landing page that introduces the tool. The goal is to easy users into the tool and make the initial experience less jarring.

- On every visit the user is redirected to the landing page (`/` -> `/start`)
- Once on the landing page the user and the user clicks "Simulate", the redirect will never happen again for that session (until a refresh or a real browser navigation happens)
- The user can decide to check the checkbox at the bottom of the page before clicking "Simulate". This will cause the landing page redirect to be permanently disabled. In addition the checkbox will not be present on the landing page if the user chooses to directly navigate to it.
  -  image

### Responsiveness
foo

### Checklist
- [x] Compress image  with https://squoosh.app/
- [x] Responsive: Breakpoints for Mobile and Tables
- [x] Checkbox to skip landing page on subsequent visits.
- [x] Less scientific introductory text section built from the [Overview section](https://github.com/neherlab/covid19_scenarios/blob/master/README.md#overview) of the readme
- [ ] Pixel push
  - See if I can do the svg curve in a smarted way. I'm not great with svg
  - fix the gradient color. It looks darker and not as good, but this is the linear gradient code Figma says corresponds to my design
  - lots more

## Related issues
- Fixes https://github.com/neherlab/covid19_scenarios/issues/107

## Testing

TODO

### CODE REVIEW

**Comment by rsharvill** [comment]
@tryggvigy I created a user guide in the README.md that I think would be helpful to include on the landing page. Let me know if you would like me to add this once this pull request is approved

File: src/components/LandingPage/LandingPage.scss [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    position: unset;
+    display: block;
+    width: 100%;
+    margin-bottom: 20px;
+  }
+
+  .landing-page__content-section {
+    padding: 20px 40px;
+  }
+}
```
Comment (tryggvigy) [inline src/components/LandingPage/LandingPage.scss]: I'm not happy with the state of this CSS. I might spend some more time to get it into better shape. e.g. not use grid, consolidate breakpoints, contain weirdness due to the svg wave better, less magic numbers, use more variables, etc.

**Comment by tryggvigy** [comment]
@ivan-aksamentov This is now functionally ready. 

Would love a review before writing the tests in case something big needs to change.

**Comment by tryggvigy** [comment]
Yeah sounds good 👍

**Pull request merged**